lib/events/homeHeaderPreEvents.js

```diff
@@ -1,5 +1,11 @@
 if (Meteor.isClient) {
 
+	function catChoice(elem) {
```

Comment:
Function declarations should not be placed in blocks. Use a function expression or move the statement to the top of the outer function.